sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -844,10 +846,6 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
       s"Cannot delete from table ${table.name} where ${filters.mkString("[", ", ", "]")}")
   }
 
-  def deleteOnlySupportedWithV2TablesError(): Throwable = {
```

Comment:
Why did? Is it not longer used after this factoring?

Reply:
We can use `operationOnlySupportedWithV2TableError(operation: String)` at L544